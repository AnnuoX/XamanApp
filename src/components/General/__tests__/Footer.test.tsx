/**
 * @format
 */

import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import { Footer } from '..';

describe('[Footer]', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Footer />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
