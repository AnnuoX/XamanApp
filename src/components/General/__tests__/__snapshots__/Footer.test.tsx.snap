// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Footer] renders correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "justifyContent": "center",
        "padding": 20,
        "width": 750,
      },
      {
        "paddingBottom": undefined,
      },
      undefined,
    ]
  }
/>
`;
